core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -310,6 +310,12 @@ oppia.directive('conversationSkin', [
             $scope.isLoggedIn = userInfo.isLoggedIn();
           });
 
+          $scope.getFeedbackPopoverUrl = function() {
+            return UrlInterpolationService.getDirectiveTemplateUrl(
+              '/pages/exploration_player/' +
+              'feedback_popup_container_directive.html');
```

Comment:
You have two copies of this method in different files, I think that in this case, it is quite okay, but at least the path should be in some kind of constant.

Reply:
Done.